README.md

```diff
@@ -16,7 +16,8 @@ Through this interface you should set the following configuration to the `Polari
 -   **loggerLevel** (_LoggerLevel_) - The level the logger is listening on, can be one of the following levels: `fatal` /
     `error` / `warn` / `info` / `debug` / `trace`.
 -   **logstashConfigurations** (_LogstashConfiguration[] - optional_) - Through this property you can set multiple logstash
-    hosts, ports and protocols (**Notice that you can use either TCP or UDP for each logstash config**).
+    hosts, ports and protocols (**Notice that you can use TCP/UDP or DYNAMIC for each logstash config**).
+    Use DYNAMIC to make the library decide which protocol to use.
```

Comment:
"the library" you mean MGF? if so write "MGF"

Reply:
but it is not mgf :(